packit/api.py

```diff
@@ -1078,6 +1092,72 @@ def get_default_commit_description(
             resolved_bugs=resolved_bugs_msg,
         )
 
+    def get_release_link(self, upstream_tag: str, version: str) -> str:
+        """
+        Get link to the release via API if the GitProject is initialised.
+        """
+        if not self.up.local_project.git_project:
+            return ""
+
+        release = self.up.local_project.git_project.get_release(
+            tag_name=upstream_tag,
+            name=version,
+        )
+        return release.url if release else ""
+
+    def get_pr_description(
+        self,
+        upstream_tag: str,
+        version: str,
+        release_monitoring_project_id: Optional[int] = None,
+        resolved_bugs: Optional[list[str]] = None,
+    ) -> str:
+        """
+        Get the description used in pull requests for syncing release.
+        """
+        resolved_bugzillas_info = ""
+        if resolved_bugs:
```

Comment:
This is unnecessary since the loop below won't be executed if `resolved_bugs` is an empty list.

Reply:
but the `resolved_bugs` can also be None, not only empty list